features/pre-post-transformation/README.md

```diff
@@ -0,0 +1,119 @@
+---
+description: Request transformations on media before or after upload.
```

Comment:
Programmatically modify media file before upload. And eagerly generate different image & video transformations right after upload so that they are available for fast delivery before your users access them for the first time.

Reply:
Done in https://github.com/imagekitio/docs-keep-a-changelog/pull/201/commits/376303b5a5e6c3e690d5cf118c9b131ea186fb65.

Note: Kept description a bit brief & expanded in the first paragraph.